src/testup/core.rb

```diff
@@ -9,7 +9,9 @@
 
 # TODO: Defer this to a point so it doesn't happen during SketchUp startup.
 require 'testup/gem_helper'
-TestUp::GemHelper.require('minitest', 'minitest', version: '5.4.3')
+
+# Minitest 5.15.0 is compatible with Ruby '>= 2.2, < 4.0', released 15-Dec-2021
```

Comment:
With that the `.rubocop.yml` should update `TargetSketchUpVersion` and `TargetRubyVersion`.

Reply:
I wasn't sure what you wanted to do, I don't think I have a complete set of old Ruby 2.2. SU versions.

Reply:
I can have a look at the rubocop config.